plugins/cpp/parser/include/cppparser/cppparser.h

```diff
@@ -53,10 +52,29 @@ class CppParser : public AbstractParser
   bool isSourceFile(const std::string& file_) const;
   bool isNonSourceFlag(const std::string& arg_) const;
   bool parseByJson(const std::string& jsonFile_, std::size_t threadNum_);
-  void worker();
+  int worker(const clang::tooling::CompileCommand& command);
```

Comment:
Follow the coding convention (Write underscore at the end of function parameters).